supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,17 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[baseos-repository]]
+==== image:images/yes.png[yes] BaseOS repository (noun)
+*Description*: RHEL content in the _BaseOS repository_ is intended to provide the core set of the underlying OS functionality that provides the foundation for all installations.
```

Comment:
```suggestion
*Description*: RHEL content in the _BaseOS repository_ provides the core set of the underlying OS functionality that provides the foundation for all installations.
```

Just a little suggestion here because the passive 'is intended to' doesn't seem to add anything meaningful. It's not clear who is the one who intends. If you keep it though, maybe it'd be better to explain it a bit more.